htdocs/core/lib/functions2.lib.php

```diff
@@ -814,7 +814,7 @@ function get_next_value($db,$mask,$table,$field,$where='',$objsoc='',$date='',$m
     //print "masktri=".$masktri." maskcounter=".$maskcounter." maskraz=".$maskraz." maskoffset=".$maskoffset."<br>\n";
 
     // Define $sqlstring
-    $posnumstart=strrpos($maskwithnocode,$maskcounter);	// Pos of counter in final string (from 0 to ...)
+    $posnumstart=mb_strrpos($maskwithnocode,$maskcounter, 'UTF-8');	// Pos of counter in final string (from 0 to ...)
```

Comment:
mb_* functions depends on the mbstrings extension that may or may not be available.
You should at least check if it's available before using it and fallback to the regular one if unavailable.
(Yeah PHP's unicode support sucks…)